server/routes/private.js

```diff
@@ -49,6 +50,37 @@ router.post('/api/book-appointment', async (req, res) => {
 
 });
 
+router.post('/api/add_recurring_schedule', async (req, res) => {
+    // Validate appropriate parameters are passed into the book appointment endpoint.
+    // Verification to make sure appointments are only made for future dates and that the worker is available during that time will be handled by the front end.
+    const paramSchema = Joi.object({
+        workerTimeslotId: Joi.number().integer().required(),
+        slotId: Joi.number().integer().required(),
+        schoolId: Joi.number().integer().required(),
+        workerId: Joi.number().integer().required(), 
+        status:Joi.string().max(300).required(),
+        date: Joi.string().allow(null).max(300)
```

Comment:
Should be of type date rather than string, i.e., use `Joi.date()`. Refer to #17 for an example use of Joi.date.